src/pages/api/account-abstraction-grants.ts

```diff
@@ -0,0 +1,114 @@
+      projectCategory: Category__c,
+      requestedAmount: Requested_Amount__c,
+      howDidYouHearAboutGrantsWave: Referral_Source__c,
+      referralSourceIfOther: Referral_Source_if_Other__c,
+      wouldYouShareYourResearch: Would_you_share_your_research__c,
+      twitter: Twitter__c,
+      github: Github_Username__c,
+      telegram: Alternative_Contact__c,
+      repeatApplicant: Repeat_Applicant__c,
+      canTheEFReachOut: Can_the_EF_reach_out__c,
+      additionalInfo: Additional_Information__c
+    } = body;
+    const { SF_PROD_LOGIN_URL, SF_PROD_USERNAME, SF_PROD_PASSWORD, SF_PROD_SECURITY_TOKEN } =
+      process.env;
+
+    const conn = new jsforce.Connection({
+      // you can change loginUrl to connect to sandbox or prerelease env.
+      loginUrl: SF_PROD_LOGIN_URL
+    });
+
+    conn.login(SF_PROD_USERNAME!, `${SF_PROD_PASSWORD}${SF_PROD_SECURITY_TOKEN}`, async err => {
+      if (err) {
+        console.error(err);
+        return resolve();
+      }
+
+      const application = {
+        FirstName: FirstName.trim(),
+        LastName: LastName.trim(),
+        Email: Email.trim(),
+        Title: Title.trim(),
+        Applying_as_a__c: Applying_as_a__c.trim(),
+        Applying_as_Other__c: Applying_as_Other__c.trim(),
+        Company: Company.trim(),
+        npsp__CompanyCountry__c: npsp__CompanyCountry__c.trim(),
+        Countries_of_Team__c: Countries_of_Team__c.trim(),
+        Time_Zone__c: Time_Zone__c.trim(),
+        Project_Name__c: Project_Name__c.trim(),
+        Project_Description__c: Project_Description__c.trim(),
+        Category__c: Category__c.trim(),
+        Requested_Amount__c: Requested_Amount__c.trim(),
+        Referral_Source__c: Referral_Source__c.trim(),
+        Referral_Source_if_Other__c: Referral_Source_if_Other__c.trim(),
+        Would_you_share_your_research__c: Would_you_share_your_research__c.trim(),
+        Twitter__c: Twitter__c.trim(),
+        Github_Username__c: Github_Username__c.trim(),
+        Alternative_Contact__c: Alternative_Contact__c.trim(),
+        Repeat_Applicant__c, // this is a boolean value, no trim applied
+        Can_the_EF_reach_out__c, // this is a boolean value, no trim applied
+        Additional_Information__c: Additional_Information__c.trim(),
+        Proactive_Community_Grants_Round__c: 'Account Abstraction 2023', // this value is hardwired, depending on the type of grant round
+        RecordTypeId: process.env.SF_RECORD_TYPE_GRANTS_ROUND! // Proactive Grant Round
+      };
+
+      // Single record creation
+      conn.sobject('Lead').create(application, async (err, ret) => {
+        if (err || !ret.success) {
+          console.error(err);
+          res.status(400).json({ status: 'fail' });
+          return resolve();
+        }
+
+        // send submission data to a google spreadsheet
+        try {
+          await addRowToSpreadsheet(
+            {
+              id: '1MUH1hUdeHpTRXEYLzpokHEV1tQYHmy83RDpl7ny-wxQ',
```

Comment:
offtopic but could we move this value to an .env var in a separated PR? 🤔

Reply:
We've decided not to include them on the `.env` file because it was exceeding the env vars size limit.